cloudfiles/interfaces.py

```diff
@@ -61,9 +66,11 @@ def __exit__(self, exception_type, exception_value, traceback):
     self.release_connection()
 
 class FileInterface(StorageInterface):
-  def __init__(self, path, secrets=None):
+  def __init__(self, path, secrets=None, user_project=None):
     super(StorageInterface, self).__init__()
     self._path = path
+    if user_project is not None:
+      raise ValueError("Setting RequestPayer for FileInterface is not supported. Must be None, got '{}'.".format(user_project))
```

Comment:
It's kind of funny that you give an S3 error for GCS terminology here. Maybe something more neutral like "Setting requester pays is not supported for the file protocol. user_project must be None. Got: {}"